### PR TITLE
NodeJS task-3 done

### PR DESCRIPTION
**Pull request by MD-Liyaqath-Ali**

### CODE REVIEW

File: NodeJS-task-3/app.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const router = require("./routes/routers.js").router;
```
Comment (leenabodhe) [inline NodeJS-task-3/app.js:2]: router folder is not present

**Review by leenabodhe — CHANGES_REQUESTED** [review]
Please add missing files

File: NodeJS-task-3/controllers/userControllers.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    }
+}
+
+//save user in db using services 
+exports.saveUser = async (req, res) => {
+    const { id, login, age, password } = req.body;
+    console.log(id);
+    const isdeleted = false;
+    try {
+        let record = await userService.saveUserInDB(id, login, age, password, isdeleted);
```
[the remark this answers is not in the document]
Comment (MD-Liyaqath-Ali) [reply NodeJS-task-3/controllers/userControllers.js]: Changes made

File: NodeJS-task-3/app.js (hunk 1 above)
Comment (MD-Liyaqath-Ali) [reply NodeJS-task-3/app.js:2]: Changes made

**Pull request merged**